docs/project/set-up-git.md

```diff
@@ -92,8 +92,9 @@ You indicate your agreement by signing your `git` commits like this:
 
 To create a signature, you configure your username and email address in Git.
 You can set these globally or locally on just your `docker-fork` repository.
-You must sign with your real name. We don't accept anonymous contributions or
-contributions through pseudonyms.
+You must sign with your real name. You can sign your git commit automatically
+with `git commit -s`. We don't accept anonymous contributions or contributions
```

Comment:
Docker does not...

@vdemeester I am trying to kill the :crown: in our docs (royal we).